rust/agents/relayer/src/msg/metadata/aggregation.rs

```diff
@@ -90,15 +90,15 @@ impl AggregationIsmMetadataBuilder {
         sub_modules: Vec<IsmAndMetadata>,
         message: &HyperlaneMessage,
         threshold: usize,
+        invalid_isms: Vec<(ModuleType, H256)>,
     ) -> Option<Vec<SubModuleMetadata>> {
         let gas_cost_results: Vec<_> = join_all(
             sub_modules
                 .iter()
                 .map(|module| module.ism.dry_run_verify(message, &(module.meta.metadata))),
         )
         .await;
-
-        // Filter out the ISMs without a gas cost estimate
```

Comment:
?

Reply:
"// Filter out the ISMs with a gas cost estimate"